### PR TITLE
Add redirect response to search api

### PR DESCRIPTION
**Pull request by jcollado**
In this PR, the redirect response is added to the search endpoint documentation.

### CODE REVIEW

File: source/includes/_search.md [hunk 1]
```diff
@@ -306,3 +311,40 @@ Notice in this example that the `count` for parent category **Baby Care Products
...
+                "<a term>",
+                "<another term>",
+            ],
+            "matched_user_segments": [
+                "<a segment>",
+                "<another segment>",
+            ]
+        }
+    }
+}
```
Comment (adetch) [inline source/includes/_search.md]: I recall seeing the below for the format - can you confirm the logic above is correct?

```json
 {
    "request": {
    ...
    },
    "response": {
        "redirect": {
                "data": {
                    "url": "<url>"
                },
                "matched_terms": [
                     "<a term>",
                     "<another term>",
                ],
                "matched_user_segments": [
                     "<a segment>",
                     "<another segment>",
                ]
            }
        }
    }
 }
```

Comment (jcollado) [reply source/includes/_search.md]: Fixed in 1c78401

File: source/includes/_search.md [hunk 2]
```diff
@@ -344,7 +344,7 @@ structure with the following attributes:
 
 Attribute | Type | Description
 --------- | -----| -----------
-data | object | Object containing the target URL together with any metadata defined in the redirect rule.
 redirect | object | Object containing information abou the match.
```
Comment (adetch) [inline source/includes/_search.md]: Object containing information about the match.

Comment (jcollado) [reply source/includes/_search.md]: Fixed in 15bc209

**Pull request merged**